src/backend/main.ts

```diff
@@ -223,13 +229,13 @@ async function createWindow(): Promise<BrowserWindow> {
   if (!app.isPackaged) {
     /* eslint-disable @typescript-eslint/ban-ts-comment */
     //@ts-ignore
-    import('electron-devtools-installer').then((devtools) => {
-      const { default: installExtension, REACT_DEVELOPER_TOOLS } = devtools
+    // import('electron-devtools-installer').then((devtools) => {
```

Comment:
Ah great catch. I was getting the following errors in the console

 ```
"Electron sandbox_bundle.js script failed to run", source: node:electron/js2c/sandbox_bundle (160)
"TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))", source: node:electron/js2c/sandbox_bundle (160)
```

which are caused by React DevTools (https://github.com/electron/electron/issues/32133#issuecomment-1107635348). Seems they have a PR in review that fixes the issue, so I guess it will be resolved shortly.

There were several issues preventing the app from starting at that point in time, so in my troubleshooting, I disabled it. It turns out it was not the cause, and the app now runs fine with it enabled. I will uncomment and push another commit. Thank you!